lib/text_message.rb

```diff
@@ -0,0 +1,30 @@
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load("data.env")
+
+class TextMessage
+
+  def send
+    send_text
+  end
+
+  private
+
+  def send_text
+    account_sid = ENV["TWILIO_ACCOUNT_SID"]
+    auth_token = ENV["TWILIO_AUTH_TOKEN"]
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+
+    @client.messages.create(
+      from: ENV["TWILIO_NUMBER"],
+      to: ENV["MY_NUMBER"],
```

Comment:
This will be the number of whoever places the order so, perhaps we need to be collecting their number when the order is created.